commands/show/cluster/command_test.go

```diff
@@ -49,22 +61,146 @@ func TestShowAWSCluster(t *testing.T) {
 		clusterID:   "cluster-id",
 		scheme:      "giantswarm",
 		authToken:   "my-token",
+		verbose:     true,
```

Comment:
Why verbose?

Reply:
I enabled it because I wanted to actually see what's happening in the test. This way the test actually covers more code lines.